src/main/java/lotto/domain/GeneratorNumber.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class GeneratorNumber {
+
+    private static final int LOTTO_NUMBERS_SIZE = 6;
+    private static final int LIMIT_MINIMUM_NUMBER = 1;
+    private static final int LIMIT_MAXIMUM_NUMBER = 45;
+    private static final List<Integer> numbers
+        = IntStream.range(LIMIT_MINIMUM_NUMBER, LIMIT_MAXIMUM_NUMBER + 1)
+        .boxed().collect(Collectors.toList());
```

Comment:
아래와 같이 변경해보면 어떨까요? 🤔 
```suggestion
    private static final List<Integer> LOTTO_NUMBERS;

    static {
        LOTTO_NUMBERS = IntStream.rangeClosed(LIMIT_MINIMUM_NUMBER, LIMIT_MAXIMUM_NUMBER)
                .boxed()
                .collect(Collectors.toList());
    }
```

Reply:
stream 구문의 경우 한 줄에 하나의 메서드만 호출하도록 해주세요! 🙏 
위와 같이 변경하게 되면, 가독성도 높일 수 있고 인텔리제이의 편의 기능(= 우측에 리턴 타입 명시)을 이용할 수 있어요. 😃


<img width="770" alt="image" src="https://user-images.githubusercontent.com/47850258/153256170-914f18ed-e8f0-451b-9c86-63bb24221326.png">

Reply:
한 줄에 하나의 메서드만을 호출하도록 변경하였습니다.